fandogh_cli/service_commands.py

```diff
@@ -168,9 +169,10 @@ def service_details(service_name):
 @click.option('-p', '--parameter', 'parameters', help='Manifest parameters', multiple=True)
 @click.option('-d', 'detach', is_flag=True, default=False,
               help='detach terminal.')
-def service_apply(file, parameters, detach):
+@click.option('--read_env', is_flag=True, default=False, help='this command will help you read os environment variables')
```

Comment:
why this option is needed?
we can always fallback to env variable if parameter does not have value